deploy/deploy_full.sh

```diff
@@ -14,12 +14,12 @@ function exit_script {
 }
 
 # Install required components
-$script_path/install.sh "deploy" || exit_script "Development components installation failed"
+$script_path/install.sh "pipeline" || exit_script "Development components installation failed"
 
 # Deploy services
-$script_path/deploy.sh "deploy" || exit_script "Development of services failed"
+$script_path/deploy.sh "pipeline" || exit_script "Development of services failed"
 
 # Deploy lock services
-$script_path/lock.sh "deploy" || exit_script "Development of lock services failed"
+$script_path/lock.sh || exit_script "Development of lock services failed"
```

Comment:
Is there a particular reason we don't have a "pipeline" flag for conditional printing of a reboot message in `lock.sh`?

Reply:
seems like a derp, we should suppress for lock as well and reboot after all 3 are complete.